lib/resources/example_json/r4_examples_charge_item.rb

```diff
@@ -513,5 +513,208 @@ module Resources
         }
       ]
     }.freeze
+
+    R4_CHARGE_ITEM_MODIFY ||= {
+      'resourceType': 'Parameters',
+      'parameter': [
+        {
+          'name': 'chargeItem',
+          'resource': {
+            'resourceType': 'ChargeItem',
+            'status': 'unknown',
+            'subject': {
+              'reference': 'Patient/12768550'
+            },
+            'context': {
+              'reference': 'Encounter/97965088'
+            },
+            'occurrenceDateTime': '2021-01-08T10:38:00Z',
+            'quantity': {
+              'value': 31
+            },
+            'code': {
+              'coding': [
+                {
+                  'extension': [
+                    {
+                      'valueCoding': {
+                        'system': 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/14002',
+                        'code': '667687'
+                      },
+                      'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/bill-code-schedule'
+                    }
+                  ],
+                  'system': 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/CodeSystem/BillCodes-CDM_SCHED',
+                  'code': '64897'
+                }
+              ]
+            },
+            'performer': [
+              {
+                'function': {
+                  'coding': [
+                    {
+                      'system': 'http://terminology.hl7.org/CodeSystem/v2-0912',
+                      'code': 'OP',
+                      'display': 'Ordering Provider'
+                    }
+                  ],
+                  'text': 'Ordering Provider'
+                },
+                'actor': {
+                  'reference': 'Practitioner/11817978'
+                }
+              }
+            ],
+            'reason': [
+              {
+                'extension': [
+                  {
+                    'valueUnsignedInt': 1,
+                    'url': 'https://fhir-ehr.cerner.com/r4/StructureDefinition/priority'
+                  }
+                ],
+                'coding': [
+                  {
+                    'extension': [
+                      {
+                        'valueCoding': {
+                          'system': 'https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/14002',
+                          'code': '615217'
```

Comment:
I believe 615217 is the value for a Revenue Code. I think 3693 is the proper code value for ICD

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/780/commits/0878c6c8ccb78566c363cdb2ff6a03fe5a21c8f9#diff-5cd46ad5f0af8a24217f67b656e77ce3624d6cefec3a18eddf7adc12bd47d35fR583